.github/workflows/test.yml

```diff
@@ -66,6 +66,12 @@ jobs:
       - name: Install drupal
         run: test-drupal-project install
 
+      - name: Get custom thunder_admin branch
+        run: |
+          rm -rf docroot/themes/contrib/thunder_admin
+          git clone --depth 1 --single-branch --branch 8.x-3.x https://github.com/BurdaMagazinOrg/theme-thunder-admin.git docroot/themes/contrib/thunder_admin
```

Comment:
Instead of hard coding th branch, we should add an environment variable, that is easily spottable and changeable